src/strategy/actions/TradeStatusAction.cpp

```diff
@@ -235,6 +237,17 @@ bool TradeStatusAction::CheckTrade()
 
     if (!botMoney && !playerMoney)
         return true;
+	
+	// Player just wants money from the bot
+    if ( ! isGettingItem && ! isGivingItem && botMoney > playerMoney )
```

Comment:
There shouldn't be all these spaces here
`if (!isGettingItem && !isGivingItem && botMoney > playerMoney)`